backend/api/Services/MissionRunService.cs

```diff
@@ -88,8 +90,10 @@ private IQueryable<MissionRun> GetMissionRunsWithSubModels()
 
         public async Task<MissionRun> Create(MissionRun missionRun)
         {
+            missionRun.Id ??= Guid.NewGuid().ToString(); // Useful for signalR messages
```

Comment:
Does this mean we set the ID instead of leaving it to the database to set the ID? Is it done so the frontend can have the ID to make requests with?

Reply:
If the signalR event is passed after `SaveChangesAsync` wouldn't `missionRun` contain the ID?

Reply:
I tihnk we would have to read it in again from the database to get the generated ID, but I can do that